### PR TITLE
Feature/weights queries

### PR DESCRIPTION
**Pull request by ArEnSc**
Seeding and creation rest of the other operations wip.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -75,3 +75,4 @@ TTS_HIFIGAN_SUPERRES_VOCODER_MODEL_FILENAME=hifigan_superres_twilight_33000.pt
 
 # Development key. Permissible to check in.
 NEWRELIC_API_KEY=c402be1c4c4216ec83d0cfc0851520cfFFFFNRAL
+STORYTELLER_ROOT="/home/tensor/code/storyteller/"
```
Comment (echelon) [inline .env:78]: I need to figure out why this is necessary :(

File: crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    pub creator_set_visibility: Visibility,
+
+    pub ietf_language_tag: String,
+    pub ietf_primary_language_subtag: String,
+
+    pub created_at: DateTime<Utc>,
+    pub updated_at: DateTime<Utc>,
+}
+
+pub async fn list_datasets_by_username(
```
Comment (echelon) [inline crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs]: nit: function name `list_weights_by_username`

File: crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    mysql_pool: &MySqlPool,
+    username: &str,
+    can_see_deleted: bool,
+) -> AnyhowResult<Vec<WeightsRecord>> {
+    let mut connection = mysql_pool.acquire().await?;
+    list_datasets_by_username_with_connection(&mut connection, username, can_see_deleted).await
+}
+
+pub async fn list_datasets_by_username_with_connection(
```
Comment (echelon) [inline crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs]: nit: function name

File: crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    pub weights_token: ZsVoiceDatasetToken,
+    pub title: String,
+
+    pub creator_user_token: UserToken,
+    pub creator_username: String,
+    pub creator_display_name: String,
+    pub creator_email_gravatar_hash: String,
+    pub creator_set_visibility: Visibility,
+
+    pub ietf_language_tag: String,
```
Comment (echelon) [inline crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs]: probably want to return more fields - weights type, weights category,  description

File: crates/lib/mysql_queries/src/queries/model_weights/update_weight.rs [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+pub struct UpdateWeightArgs<'a> {
+    pub weight_token: &ModelWeightToken,
+    pub title: Option<&'a str>,
+    pub description_markdown: &'a str,
+    pub description_rendered_html: &'a str,
+    pub creator_set_visibility: &'a Visibility,
+    pub mysql_pool: &'a MySqlPool,
+}
+
+pub async fn update_voice(args: UpdateWeightArgs<'_>) -> AnyhowResult<()> {
```
Comment (echelon) [inline crates/lib/mysql_queries/src/queries/model_weights/update_weight.rs]: nit: function name

File: .env (hunk 1 above)
Comment (ArEnSc) [reply .env:78]: Oh you might have missed my explainer, I generally always run a git reset --hard origin/master to ensure there are no caching issues, so it's absolutely fine ill remove it later ~

File: crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs (hunk 2 above)
Comment (ArEnSc) [reply crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs]: these are the template to the work haha will fix soon =)

File: crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs (hunk 3 above)
Comment (ArEnSc) [reply crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs]: above as well

File: crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs (hunk 4 above)
Comment (ArEnSc) [reply crates/lib/mysql_queries/src/queries/model_weights/list_weights_by_user.rs]: yes

File: crates/lib/mysql_queries/src/queries/model_weights/update_weight.rs (hunk 5 above)
Comment (ArEnSc) [reply crates/lib/mysql_queries/src/queries/model_weights/update_weight.rs]: you are too kind ... not a nit haha theres a couple of passes I go over I am more focused on functionality right now

**Pull request merged**